docs/api/speech_challenges.md

```diff
@@ -161,19 +168,19 @@ Location: https://api.itslanguage.io/challenges/speech/4
 PUT /challenges/speech/:challenge HTTP/1.1
 ```
 
-* `challenge` - **Required** The speech challenge identifier.
+- `challenge` - **Required** The speech challenge identifier.
 
 Update one or more properties of an existing speech challenge.
 
 ### Request parameters
 
-Name             | Type     | Description
------------------|----------|------------
-`topic`          | `string` | **Optional** A question or topic serving as guidance to the student.
-`referenceAudio` | `blob`   | **Optional** The reference audio fragment.
-`srt`            | `blob`   | **Optional** The transcription of the challenge in `.srt` format.
-`image`          | `blob`   | **Optional** An image to show with the challenge.
-
+| Name             | Type     | Description                                                          |
+| ---------------- | -------- | -------------------------------------------------------------------- |
+| `topic`          | `string` | **Optional** A question or topic serving as guidance to the student. |
```

Comment:
Topic is not optional, even if you have an srt its-ws-server will not work without a topic.